src/modules/secureStorage.json

```diff
@@ -0,0 +1,265 @@
+  },
+  "methods": [
+    {
+      "name": "get",
+      "summary": "Get stored value by key",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-uses": [
+            "xrn:firebolt:capability:secure:storage"
+          ]
+        }
+      ],
+      "params": [
+        {
+          "name": "scope",
+          "summary": "The scope of the key/value",
+          "schema": {
+            "$ref": "#/components/schemas/StorageScope"
+          },
+          "required": true
+        },
+        {
+          "name": "key",
+          "summary": "Key to get",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        }
+      ],
+      "result": {
+        "name": "value",
+        "summary": "The retrieved value, if found.",
+        "schema": {
+          "type": "string"
+        }
+      },
+      "examples": [
+        {
+          "name": "Successfully retrieve a refresh token with key authRefreshToken",
+          "params": [
+            {
+              "name": "scope",
+              "value": "device"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            }
+          ],
+          "result": {
+            "name": "value",
+            "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
+          }
+        },
+        {
+          "name": "Attempt to retrieve a a key with no value set",
+          "params": [
+            {
+              "name": "scope",
+              "value": "account"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            }
+          ],
+          "result": {
+            "name": "value",
+            "value": null
+          }
+        }
+      ]
+    },
+    {
+      "name": "set",
+      "summary": "Set or update a secure data value",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-uses": [
+            "xrn:firebolt:capability:secure:storage"
+          ]
+        }
+      ],
+      "params": [
+        {
+          "name": "scope",
+          "summary": "The scope of the data key",
+          "schema": {
+            "$ref": "#/components/schemas/StorageScope"
+          },
+          "required": true
+        },
+        {
+          "name": "key",
+          "summary": "Key to set",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "value",
+          "summary": "Value to set",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "options",
+          "summary": "Optional parameters to set",
+          "schema": {
+            "$ref": "#/components/schemas/StorageOptions"
+          },
+          "required": false
+        }
+      ],
+      "result": {
+        "name": "success",
+        "schema": {
+          "type": "boolean"
```

Comment:
We should not have success booleans. If something doesn't succeed it will get an error in the catch block.